### ISSUE TITLE
Twitter Stream agent - add include_retweet as an option

### ISSUE THREAD

**Issue by zoomequipd** [huginn/huginn#2150]
Would it be possible to make this an option in the agent much like the Twitter User agent has a include_retweet option? 

My cursory look at the issue seems that retweets are silently discarded on due to below reference if statement.

https://github.com/huginn/huginn/blob/6a1509d9d3c7d3ee5911e1d5b15a8b8b75528ab2/app/models/agents/twitter_stream_agent.rb#L225

**Comment by dsander** [comment]
Sure that should not be too hard to do, would you be interested in working on it?

**Comment by zoomequipd** [comment]
Yeah, I can handle that.  If you want assign the issue to me and I can take a look at this next week.
